### ISSUE TITLE
Preview2 fails

### ISSUE THREAD

**Issue by enricosada** [dotnet/netcorecli-fsc#12]
Using dotnet sdk preview2 `1.0.0-preview2-003112`

Repro:

```
dotnet new --lang f#
dotnet restore
dotnet build
```

Error:

```
 internal error: Could not load type 'System.Reflection.PortableExecutable.ContentId' from assembly 'System.Reflection.Metadata, Version=1.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
```

with `dotnet-compile-fsc` restored: 

```
      "dotnet-compile-fsc/1.0.0-preview2-003071": {
        "type": "package",
        "dependencies": {
          "Microsoft.DotNet.Cli.Utils": "1.0.0-rc2-002678",
          "Microsoft.DotNet.Compiler.Common": "1.0.0-rc2-002678",
          "Microsoft.DotNet.ProjectModel": "1.0.0-rc4-003139",
          "Microsoft.FSharp.Compiler.netcore": "1.0.0-alpha-160509",
          "Microsoft.NETCore.App": "1.0.0"
        },
```

**Comment by eerhardt** [comment]
@enricosada - there was a breaking change in System.Reflection.Metadata during Ask Mode of .NET Core 1.0:

https://github.com/dotnet/corefx/commit/a9442a6fe0d0d616ce372018c86ff435cf5337c2

`Microsoft.FSharp.Compiler.netcore` is going to have to be recompiled (and fixed) on the latest contracts in order to fix this up.

@ericstj @tmat @KevinRansom

**Comment by shanselman** [comment]
Who is on this?

**Comment by KevinRansom** [comment]
I need to rebuild fsharp for the latest coreclr.  I have been having connection difficulties.  But I am on it.

**Comment by ericstj** [comment]
https://github.com/dotnet/cli/pull/3627 /cc @piotrpMSFT @enricosada

**Comment by enricosada** [comment]
@KevinRansom check pr https://github.com/Microsoft/visualfsharp/pull/1290, i started to fix deps and build tool now use RTM version of .NET Core (with preview2 of sdk)

**Comment by enricosada** [comment]
fixed by version `1.0.0-preview2-003131` of .net core sdk (ref [announcement](https://blogs.msdn.microsoft.com/dotnet/2016/09/13/announcing-september-2016-updates-for-net-core-1-0/) )
